Sources/Xcodeproj/generate().swift

```diff
@@ -93,10 +93,15 @@ public func generate(outputDir: AbsolutePath, projectName: String, graph: Packag
         print("</plist>")
     }
 
+    let invalidModuleNames = Set(["Modules", "Headers", "Versions"])
     for module in graph.modules where module.isLibrary {
         ///// For framework targets, generate module.c99Name_Info.plist files in the 
         ///// directory that Xcode project is generated
         let name = module.infoPlistFileName
+        if invalidModuleNames.contains(name) {
```

Comment:
This is scheme generation, we should do check and warn from the method in `pbxproj().swift`
This check should be on the module.c99name (because that becomes the productname)